tests/Resources/Admin/ContentAdmin.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2017 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrinePHPCRAdminBundle\Tests\Resources\Admin;
+
+use Doctrine\Bundle\PHPCRBundle\Form\DataTransformer\DocumentToPathTransformer;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\CoreBundle\Form\Type\CollectionType;
+use Sonata\DoctrinePHPCRAdminBundle\Admin\Admin;
+use Sonata\DoctrinePHPCRAdminBundle\Tests\Resources\Document\Content;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class ContentAdmin extends Admin
+{
+    /**
+     * @var ManagerRegistry
+     */
+    private $managerRegistry;
+
+    public function setManagerRegistry(ManagerRegistry $managerRegistry)
+    {
+        $this->managerRegistry = $managerRegistry;
+    }
+
+    public function getExportFormats()
+    {
+        return array();
+    }
+
+    public function toString($object)
+    {
+        return $object instanceof Content && $object->getTitle()
+            ? $object->getTitle()
+            : $this->trans('link_add', array(), 'SonataAdminBundle');
+    }
+
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper
+            ->addIdentifier('id')
+            ->add('title');
+    }
+
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('form.group_general')
+            ->add('name', 'Symfony\Component\Form\Extension\Core\Type\TextType')
+            ->add('title', 'Symfony\Component\Form\Extension\Core\Type\TextType')
+            ->add(
+                'children',
+                CollectionType::class,
+                array('label' => false, 'type_options' => array('delete' => true, 'delete_options' => array('type' => CheckboxType::class, 'type_options' => array('required' => false, 'mapped' => false)))),
```

Comment:
yea but thats just style. I do not want to add those options at all, cause they are default. `AdmintType` of sonata creates `_delete` field of type `checkbox` instead of `CheckboxType` So without forcing my "default" option to the correct type, my `FormRegistry` flames not to know a form of type `checkbox`